did/stats.py

```diff
@@ -22,6 +22,7 @@ class Stats(object):
     option = None
     dest = None
     parent = None
+    skip = None
```

Comment:
This should not be needed. The `skip` option belongs to configuration and it doesn't have to be defined here.